src/components/DatePicker/index.jsx

```diff
@@ -37,26 +37,34 @@ export default class DatePickerComponent extends React.Component {
     onChange: PropTypes.func,
     onFocus: PropTypes.func,
     query: PropTypes.object,
+    required: PropTypes.bool,
```

Comment:
Currently, all the apps follow the isRequired naming convention - any specific reason you changed this here?

Reply:
I just meant to follow react-datepicker convention. I'll change it to `required={isRequired}`